### ISSUE TITLE
What to do after performing meta-learning?

### ISSUE THREAD

**Issue by nguyentthong** [copenlu/X-MAML#2]
Dear @nooralahzadeh , I'm really interested in your idea of applying meta-learning for cross-lingual NLU tasks.

I have a question related to your original paper "Zero-shot cross-lingual transfer with meta learning". In the paper you said that after getting the model to finish the meta-learning process, we can either do the zero-shot testing or "fine-tune the model parameters using the development set". I'm not quite clear about this. Do you mean that you continue to train the model on the validation set?

Best regards,
Thong

**Comment by nooralahzadeh** [comment]
Hi @thongnguyen050999 , 
Thanks for reaching out and your interest in our work.
In fact, if you want to do fine-tuning, after meta-learning, you can fine-tune your model using the validation set of the target language/s and then evaluate on the test set.

Please don't hesitate to contact me if you need more help.

Best,
Farhad
